api/message.go

```diff
@@ -217,6 +231,21 @@ func (c *Client) SendEmbed(
 	})
 }
 
+// SendEmbedReply posts an Embed reply to a message ID in a guild text or DM channel.
+//
+// If operating on a guild channel, this endpoint requires the SEND_MESSAGES
+// permission to be present on the current user.
+//
+// Fires a Message Create Gateway event.
+func (c *Client) SendEmbedReply(
+	channelID discord.ChannelID, e discord.Embed, msgID discord.MessageID) (*discord.Message, error) {
```

Comment:
`msgID` should be changed to something more descriptive like `refID` or preferably `referenceID`.